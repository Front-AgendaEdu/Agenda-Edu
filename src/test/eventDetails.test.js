const mockedNavigate = jest.fn();

jest.mock('react-native-vector-icons/EvilIcons', () => 'Icon')

jest.mock("@react-native-community/async-storage", () =>
	require("@react-native-community/async-storage/jest/async-storage-mock"),
);

jest.mock('@react-navigation/native', () => {
	return {
		...jest.requireActual('@react-navigation/native'),
		useNavigation: () => ({
			navigate: mockedNavigate,
		}),
	};
});



import React from 'react';
import { create, act } from 'react-test-renderer';
import renderer from 'react-test-renderer'

const tree = create(<eventDetails/>)

describe('##SNAPSHOT##', () => {
	it('Events Snapshot', () => {
		const snap = renderer.create(<eventDetails />).toJSON()
		expect(snap).toMatchSnapshot()
	})
})
