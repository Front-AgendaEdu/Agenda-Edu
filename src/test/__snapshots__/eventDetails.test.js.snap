// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`##SNAPSHOT## Events Snapshot 1`] = `<eventDetails />`;
